encoders/nlp/TransformerTorchEncoder/tests/test_transformertorchencoder.py

```diff
@@ -193,17 +189,19 @@ def test_wrong_layer_index(test_metas, layer_index):
     encoder.layer_index = layer_index
     test_data = np.array(['it is a good day!', 'the dog sits on the floor.'])
     with pytest.raises(ValueError):
-        encoded_data = encoder.encode(test_data)
+        _ = encoder.encode(test_data)
```

Comment:
Any specific reason to remove the `save_config` and `assert` part?

Reply:
because there is another test doing that

Reply:
ok, i see.